lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -692,9 +691,8 @@ monitorStakePools tr gp nl DBLayer{..} =
         -- #2196: We need the try. Arguably we shouldn't need it.
         liftIO (try @SomeException (timeInterpreter nl (epochOf currentSlot))) >>= \case
             Left _ -> return ()
-            Right currentEpoch -> do
-                let subtractTwoEpochs = epochPred <=< epochPred
-                forM_ (subtractTwoEpochs currentEpoch) $ \latestRetirementEpoch -> do
+            Right currentEpoch | currentEpoch >= 2 -> do
```

Comment:
Came to think about: isn't there an otherwise case missing now?

Reply:
Yes - oops.